ui/build.gradle.kts

```diff
@@ -35,8 +35,8 @@ plugins {
 
 node {
   download.set(true)
-  version.set("16.14.2")
-  npmVersion.set("7.24.2")
+  version.set("19.6.1")
```

Comment:
since we are not really actively developing the UI, should we maybe use the LTS version instead here?
```
Latest LTS Version: 18.14.1 (includes npm 9.3.1)
```

Reply:
WFM